docs: bluetooth: mesh: Add documentation for models shell commands (#7111)
nrfconnect/sdk-nrf: doc/nrf/libraries/bluetooth_services/mesh/gen_plvl_cli.rst

The Generic Power Level Client may send messages to both the Generic Power Level Server and the Generic Power Level Setup Server, as long as it has the right application keys.

Extended models
****************

None.

Persistent storage
*******************

None.

Shell commands
**************

The Bluetooth mesh shell subsystem provides a set of commands to interact with the Generic Power Level Client model instantiated on a device.

To make these commands available, enable the following Kconfig options:

* :kconfig:`CONFIG_BT_MESH_SHELL`
* :kconfig:`CONFIG_BT_MESH_SHELL_PLVL_CLI`

mdl_plvl instance get-all
	Print all instances of the Generic Power Level Client model on the device.

mdl_plvl instance set <elem_idx>

> 	Select the Generic Power Level Client model instantiated on the specified element id.
> 	This instance will be used in message sending.
> 	If no model instance was selected, the first model instance found on the device will be used by default.

Reviewer: ```suggestion
	Select the Generic Power Level Client model instantiated on the specified element ID.
	This instance will be used in message sending.
	If no model instance is selected, the first model instance found on the device will be used by default.
```

> 	* ``elem_idx``: Element index where to find the model instance.

Reviewer: ```suggestion
	* ``elem_idx``: Element index where the model instance is found.
```

mdl_plvl get
	Get the Power Level of the bound server.

> mdl_plvl set <lvl> [transition_time_ms  [delay_ms]]

Reviewer 2: Extra space in signature here and below

> 	Set the Power Level of a server and wait for a response.

Reviewer: ```suggestion
	Set the Power Level of the server and wait for a response.
```

	* ``lvl``: Power level value to set.
	* ``transition_time_ms``: If present, defines the transition time in the message in milliseconds.
	* ``delay_ms``: If present, defines the delay in the message in milliseconds.

mdl_plvl set-unack <lvl> [transition_time_ms  [delay_ms]]

> 	Set the Power Level of a server withour requesting a response.

Reviewer: ```suggestion
	Set the Power Level of the server without requesting a response.
```
